docs/public/installing-odo.adoc

```diff
@@ -15,6 +15,15 @@ v2.0.0-alpha-2 of odo uses **devfile** as its default deployment mechanism.
 # chmod +x /usr/local/bin/odo
 ----
 
+== Installing odo on Linux on IBM Z
+
+=== Binary installation
+
+----
+# curl -L https://github.com/openshift/odo/releases/download/v2.0.0-alpha-2/odo-linux-s390x -o /usr/local/bin/odo
```

Comment:
maybe we need say `take v2.0.0-alpha-2` as example?
should we wait for release otherwise we need update it every time (or at least we need keep an eye on every release change ...)

Reply:
@jichenjc The context is `Installing v2.0.0-alpha-2 of odo` . We just keep along with upstream.

Reply:
should we update to beta now - or maybe just wait until the v2 is released? so we dont have to update the docs multiple times?

Reply:
@girishramnani  What's the branch for v2, master? If so, I think update the version where upstream is.